sdk/dotnet/README.md

```diff
@@ -0,0 +1,56 @@
+# Experimental .NET Language Provider
```

Comment:
Note: do not review this yet.  this is what Matt wrote.  I have no idea if it is applicable in my PR.

Reply:
Should we remove this if it is not true?